lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -74,9 +73,8 @@ def normalize_results
 
           normalize_end_dt = Time.now.utc
           @normalize_time_s = normalize_end_dt - normalize_start_dt
-          @normalized_size = results.to_s.size if performance_data?
           Rails.logger.info("Time to normalize data: #{normalize_time_s}s")
-          results = append_performance_data(results) if performance_data?
+          results = append_data_outside_results(results)
```

Comment:
Generalizing the name of this method so it can be used later to add other types of data to the results.